CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Unreleased
 
 #### Changed
 
+-	Added new columns in summary tables (\#365)
```

Comment:
Align with the rest of the text (this is more indented) and add sentence punctuation (i.e., a period at teh end) like the other entries.